webpack.config.js

```
@@ -1,9 +1,29 @@
 /* eslint no-param-reassign: 0 */
 // This config is for building dist files
 const getWebpackConfig = require('antd-tools/lib/getWebpackConfig');
-
 const { webpack } = getWebpackConfig;
 
+// Add PostCSS support
+const addPostCssLoader = (config) => {
+  // Find CSS rule and modify it
+  config.module.rules.forEach(rule => {
```

ESLint

[error] Expected parentheses around arrow function argument.

(arrow-parens)